### PR TITLE
Add strings.is_blank

### PR DESCRIPTION
**Pull request by anderseknert**
And make lines() work with Windows style newlines.

### CODE REVIEW

File: lib/strings.rego [hunk 1]
```diff
@@ -48,3 +48,6 @@ pad_right(s, length, char) = s {
 } else = r {
 	r := sprintf("%v%v", [s, repeat(char, length - count(s))])
 }
+
+# True if string is empty or contains only whitespace characters
+is_blank(s) = regex.match(`^$|^\s+$`, s)
```
Comment (srenatus) [inline lib/strings.rego:53]: I keep forgetting that backticks exist in Rego. 💡

File: lib/strings.rego [hunk 2]
```diff
@@ -27,7 +27,7 @@ repeat(str, x) = r {
 
 # Returns a list of lines from the provided string
 lines(str) = r {
-	r := split(str, "\n")
+	r := regex.split(`\n|\r\n`, str)
 }
```
Comment (srenatus) [inline lib/strings.rego:31]: [nit] could be `lines(str) = regex.split(`\n|\r\n`, str)` if we wanted to keep style consistent and brief. But this is fine, too. 😃

**Review by srenatus — APPROVED** [review]
LGTM

File: lib/strings.rego (hunk 2 above)
Comment (anderseknert) [reply lib/strings.rego:31]: Right, I think we have a lot of functions to rewrite to that style (which I agree is preferable). Will fix in next PR, thanks! :)

**Pull request merged**